photon-client/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -57,6 +57,8 @@ class GameEstimatorTest extends SparkTestUtils with GameTestUtils {
    *       is specified in [[GameParams.featureShardIdToInterceptMap]]. This happens in
    *       [[GameDriver.prepareFeatureMapsDefault]], and there only. Here, we have to setup an intercept manually,
    *       otherwise [[GameEstimator]] learns only a dependence on the features.
+   *
+   * KEEP THIS DIDACTIC EXAMPLE
```

Comment:
Well - I'd like to keep this, so that this test is not removed later, or changed, and we lose the pedagogical aspect of this code. It is extremely useful to follow step-by-step to see how to set up the `GameEstimator`, and we don't have an equivalent anywhere else. The other places are much harder to read.